cobalt/media/base/sbplayer_bridge.cc

```diff
@@ -731,6 +731,13 @@ void SbPlayerBridge::CreatePlayer() {
 
   is_creating_player_ = true;
 
+  if (output_mode_ == kSbPlayerOutputModeInvalid) {
```

Comment:
We can move this check to the ctor, right after the call to ComputeSbPlayerOutputMode().

It's ok to have the check here, but we'd have to maintain all states.  For example, we should set `is_creating_player_` to false on return.

Reply:
As `SbPlayerBridge::TryToSetPlayerCreationErrorMessage()` needs `is_creating_player_ == True` to log error message, do we want to left `error_message = player_bridge_->GetPlayerCreationErrorMessage();` empty in this case?

Reply:
In that case, let's keep the code here and set `is_creating_player_` to false before return (after calling PlayerErrorCB).

Reply:
Fixed